examples/shopping_ads/add_performance_max_retail_campaign.py

```diff
@@ -245,16 +245,20 @@ def _create_performance_max_campaign_operation(
     # Setting a portfolio bidding strategy by resource name is not supported.
     # Max Conversion and Max Conversion Value are the only strategies supported
     # for Performance Max campaigns.
+    campaign.bidding_strategy_type = (
+        client.enums.BiddingStrategyTypeEnum.MAXIMIZE_CONVERSION_VALUE
+    )
```

Comment:
I actually don't think this need to be set at all, we should delete these three lines.

Reply:
that's correct, Ben. I tested this and the pmax campaign was created successfully with max conv value as the bidding strategy. I will delete these 3 lines, and add a new commented out line showing how to correctly set up the campaign if you want max conv instead of max conv value.